package.json

```diff
@@ -9,6 +9,7 @@
     "test": "testem",
     "start": "webpack-dev-server --hot",
     "prepublish": "npm run build",
+    "postinstall": "npm run build",
```

Comment:
Can you remove this please. Builds execute before publishing. We don't want to do this after installing.